### ISSUE TITLE
Add option to disable URL Autocomplete (like the option in Firefox Focus)

### ISSUE THREAD

**Issue by jamielinux** [mozilla-mobile/firefox-ios#7146]
### Why/User Benefit/User Problem

In **_Firefox Focus_**, there is an option to disable **URL Autocomplete**. This seems to autocomplete popular websites once you start typing in the URL bar, and I almost never find it helpful.

In **_Firefox for iOS_**, you can't disable this behaviour. It would be much appreciated if someone could implement this option! :+1: 

### Acceptance Criteria (how do I know when I’m done?)

When there is a toggle in the Search settings to enable/disable URL Autocomplete.

**Comment by yoasif** [comment]
The Focus option (at least on Android - sorry) is "For top sites" with the description: "Enable to have Firefox Focus autocomplete over 450 popular URLs in the address bar".

**Comment by Karlstens** [comment]
Firefox Suggest in the iOS Address Bar is an absolute pain on iPhone. 

When I type a word in the FireFox iOS address bar, I'm only needing it to return URLs with matching bookmark tags to the word I've typed, or matching URLs that contain the typed word found in my history, sorted by either the last time I visited that site or alternatively, how often I visited that site.

At the moment, I start typing a word in the address bar and it prefills the URL address bar with sites URLS that;

- I never want to visit
- Have no context to what I'm searching/typing in
- Are completely offtopic / unrelated to my typed word, other than they coincidentally contain the typed word in their URL.

A general example would be, I type "News" in my iOS address bar, but instead of returning my favourite News site that I visit most days (as Firefox does on my desktop), on the iOS it'll return all sorts of unrelated/unwanted sites such as www.NewSimpleRecipies.com ... every, single... time. 

In summary - The iOS URL experience must mimic the same behaviour (and configuration) found on FireFox desktop experience. There shouldn't be any difference when comparing the behaviour of the address bar on my FireFox PC vs FireFox iOS for when I type in a word.

**Comment by Dennismn** [comment]
It's very annoying that when you type in the address bar, the Firefox suggest autofills with bookmark suggestions. I don't understand why this is necessary or why we dont have the option to turn it off. Could we please have the option to turn it off? It's also not good for privacy reasons. Why assume that people want their bookmark and autofill suggestions ALWAYS on?

**Comment by jamielinux** [comment]
> It's very annoying that when you type in the address bar, the Firefox suggest autofills with bookmark suggestions. I don't understand why this is necessary or why we dont have the option to turn it off. Could we please have the option to turn it off? It's also not good for privacy reasons. Why assume that people want their bookmark and autofill suggestions ALWAYS on?

@Dennismn Autocompleting bookmarks is a different from what we're discussing here. We're discussing autocompletion of "popular websites" (as determined by Firefox). Please open a separate issue :relaxed:

**Comment by Dennismn** [comment]
Ok, thanks for letting me know. I've made a separate issue.
